LDAR_Sim/src/virtual_world/sites.py

```diff
@@ -152,6 +152,15 @@ def _create_equipment_groups(
                         site_equipment_group[1:],
                     )
                 )
+        elif equipment_groups == 0:
```

Comment:
Adding this in as a new else if is unnecessary.  The bottom else: statement already handles the default case. Instead I recommend to add a check to the handling if an integer/float is provided for equipment groups to ensure it is non-zero. This will make the default case trigger when equipment groups is 0. Better yet, would be to the above change but change the else to and elif to handle the 0 case and add a new else case to handle anything else by exiting and printing an error message stating invalid input for equipment groups and printing the offending value.